README.md

```diff
@@ -417,6 +417,10 @@ this behavior, we can use the `-I` (or `--no-ignore`) option:
 target/debug/deps/libnum_cpus-f5ce7ef99006aa05.rlib
 ```
 
+*fd* respects the global git-ignore file as well. But as said above: Only when the directory belongs to a git repository.
```

Comment:
maybe add a link to the Git documentation ("global git-ignore file").

Wording: "*fd* also respects global git-ignore files when searching inside a Git repository."